external/radiation/python/radiation_clouds.py

```diff
@@ -600,14 +600,6 @@ def progcld2(
                     qcwat[i, k] = 0.0
                     clw2[i, k] = clwf[i, k]
 
-        # -# Call module_microphysics::rsipath2(), in Ferrier's scheme, to
-        # compute layer's cloud liquid, ice, rain, and snow water condensate
-        # path and the partical effective radius for liquid droplet, rain drop,
-        # and snow flake.
-        cwp, cip, crp, csp, rew, rer, res, rsden = rsipath2(
```

Comment:
If this is not used, what about the surrounding code blocks? It would be less confusing to delete the whole block of unused code rather than just this subroutine call which happened to give the linting error.